pkg/http/api.go

```diff
@@ -2020,6 +2026,248 @@ func (a *api) onPublish(reqCtx *fasthttp.RequestCtx) {
 	}
 }
 
+type bulkPublishMessageEntry struct {
+	EntryID         string            `json:"entryID,omitempty"`
+	Event           interface{}       `json:"event"`
+	DataContentType string            `json:"dataContentType"`
+	Metadata        map[string]string `json:"metadata,omitempty"`
+}
```

Comment:
Can we reuse this from contrib?

Reply:
No, the type of Event is different ... it is interface here whereas it is []byte in contrib....